### PR TITLE
Changed REST client to MP Rest Client

### PR DESCRIPTION
**Pull request by justineechen**
- Changed Client to use Microprofile Rest Client.
- Moved private methods from InventoryManager into InventoryUtils to avoid confusion for the user when they are going through the guide.
- InventoryHealth still uses previous Client

### CODE REVIEW

File: finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java [hunk 1]
```diff
@@ -44,7 +44,8 @@ public Response getPropertiesForHost(@PathParam("hostname") String hostname) {
 
     if (!inventoryConfig.isInMaintenance()) {
       // tag::config-port[]
-      Properties props = manager.get(hostname, inventoryConfig.getPortNumber());
+      // Properties props = manager.get(hostname, inventoryConfig.getPortNumber());
```
Comment (JunyiSun) [inline finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java]: Pls remove commented code.

File: start/src/main/java/io/openliberty/guides/inventory/InventoryResource.java [hunk 2]
```diff
@@ -44,7 +44,8 @@ public Response getPropertiesForHost(@PathParam("hostname") String hostname) {
 
     if (!inventoryConfig.isInMaintenance()) {
       // tag::config-port[]
-      Properties props = manager.get(hostname, inventoryConfig.getPortNumber());
+      // Properties props = manager.get(hostname, inventoryConfig.getPortNumber());
```
Comment (JunyiSun) [inline start/src/main/java/io/openliberty/guides/inventory/InventoryResource.java]: same here. pls remove this line

**Comment by JunyiSun** [comment]
@justineechen  Please format the code using two spaces, not one tab.

**Comment by evelinec** [comment]
@JunyiSun can you take a look at the latest commits. Thanks.

File: finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java [hunk 3]
```diff
@@ -10,24 +10,22 @@
...
 @ApplicationScoped
-// end::ApplicationScoped[]
 public class InventoryManager {
 
   private InventoryList invList = new InventoryList();
-  private SystemClient systemClient = new SystemClient();
+  private InventoryUtils invUtils = new InventoryUtils();
+
+  public Properties get(String hostname, int portNumber) {
+    Properties properties = invUtils.getProperties(hostname, portNumber);
```
Comment (Kubik42) [inline finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java]: how come you need a portNumber here? I dont recall it being necessary in the other guides?

Comment (JunyiSun) [reply finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java]: The config guide uses it, and the health guide build on top of the config guide, so it's there in the first place.

Comment (Kubik42) [reply finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java]: ah ok

**Pull request merged**